docs/cpp/cpp-refactoring.md

```diff
@@ -33,6 +33,16 @@ If you do not have any precedent of where you create definitions or declarations
 
 For templates, if a function template is declared in a header file, the definition of that function template will be created in the same header file. This also applies for nontemplate member functions of class templates.
 
+### Copy declarations or definitions
+
+If you want to choose the location that your declaration or definition is added in code, you can use the code action **Copy Declaration/Definition**. This adds the declaration or definition to your clipboard rather than directly to your code.
+
+To invoke the code action, select a function that has a Quick Fix available, then select the Code Action (light bulb) and choose **Copy definition of ‘YourFunctionName’** or **Copy declaration of ‘YourFunctionName’**.
+
+![Copy a declaration or definition](images/refactoring/copy-declaration-definition.gif)
+
+If the declaration/definition is not formatted when you paste, turn on auto formatting using **Settings** (`kb(workbench.action.openSettings)`) > **Format On Paste**. Enabling this will automatically format all items you paste in the editor.
```

Comment:
This formatting issue is fixed with 1.20.0 so it might be better to omit.

Reply:
removed this section